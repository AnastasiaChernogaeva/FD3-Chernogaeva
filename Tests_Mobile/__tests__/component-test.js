
import React from 'react';
import renderer from 'react-test-renderer';
 
/*import  '../App.js';*/
import MobileCompany from '../components/mobile';


test('работа MobileCompany', () => {
 
  // создаём тестовую версию компонента
  const component = renderer.create(
    <MobileCompany />
  );
 
  // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
 
  // найдём в вёрстке компонента сами кнопки
 
 
  
  const buttonElems = component.root.find( el =>(el.props.bb=="block")); 
  // находим все кнопки
 
  buttonElems.props.forEach(elem=>{elem.props.onClick(); // по очереди нажимаем на кнопки и делаем снимк(так два раза)
  
  // получаем уже изменённый снэпшот
  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
  // и "нажмём" на неё
 
  elem.props.onClick();
  
  // получаем уже изменённый снэпшот
  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
 
})
})  
 
  
 