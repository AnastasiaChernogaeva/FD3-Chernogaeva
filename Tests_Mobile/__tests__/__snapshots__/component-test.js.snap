// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа MobileCompany 1`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Иванов 
        </td>
        <td
          className="MobileClientFIO"
        >
          Иван
        </td>
        <td
          className="MobileClientFIO"
        >
          Иванович
        </td>
        <td
          className="MobileClientBalance"
        >
          200
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Сидоров .
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидор
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидорович
        </td>
        <td
          className="MobileClientBalance"
        >
          250
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Петров 
        </td>
        <td
          className="MobileClientFIO"
        >
          Петр
        </td>
        <td
          className="MobileClientFIO"
        >
          Петрович
        </td>
        <td
          className="MobileClientBalance"
        >
          180
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 2`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Иванов 
        </td>
        <td
          className="MobileClientFIO"
        >
          Иван
        </td>
        <td
          className="MobileClientFIO"
        >
          Иванович
        </td>
        <td
          className="MobileClientBalance"
        >
          200
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Сидоров .
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидор
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидорович
        </td>
        <td
          className="MobileClientBalance"
        >
          250
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Петров 
        </td>
        <td
          className="MobileClientFIO"
        >
          Петр
        </td>
        <td
          className="MobileClientFIO"
        >
          Петрович
        </td>
        <td
          className="MobileClientBalance"
        >
          180
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 3`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Иванов 
        </td>
        <td
          className="MobileClientFIO"
        >
          Иван
        </td>
        <td
          className="MobileClientFIO"
        >
          Иванович
        </td>
        <td
          className="MobileClientBalance"
        >
          200
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Сидоров .
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидор
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидорович
        </td>
        <td
          className="MobileClientBalance"
        >
          250
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Петров 
        </td>
        <td
          className="MobileClientFIO"
        >
          Петр
        </td>
        <td
          className="MobileClientFIO"
        >
          Петрович
        </td>
        <td
          className="MobileClientBalance"
        >
          180
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 4`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Иванов 
        </td>
        <td
          className="MobileClientFIO"
        >
          Иван
        </td>
        <td
          className="MobileClientFIO"
        >
          Иванович
        </td>
        <td
          className="MobileClientBalance"
        >
          200
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Сидоров .
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидор
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидорович
        </td>
        <td
          className="MobileClientBalance"
        >
          250
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Петров 
        </td>
        <td
          className="MobileClientFIO"
        >
          Петр
        </td>
        <td
          className="MobileClientFIO"
        >
          Петрович
        </td>
        <td
          className="MobileClientBalance"
        >
          180
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 5`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Иванов 
        </td>
        <td
          className="MobileClientFIO"
        >
          Иван
        </td>
        <td
          className="MobileClientFIO"
        >
          Иванович
        </td>
        <td
          className="MobileClientBalance"
        >
          200
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Сидоров .
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидор
        </td>
        <td
          className="MobileClientFIO"
        >
          Сидорович
        </td>
        <td
          className="MobileClientBalance"
        >
          250
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Петров 
        </td>
        <td
          className="MobileClientFIO"
        >
          Петр
        </td>
        <td
          className="MobileClientFIO"
        >
          Петрович
        </td>
        <td
          className="MobileClientBalance"
        >
          180
        </td>
        <td
          className="active"
        >
          active
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 6`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 7`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
</div>
`;

exports[`работа MobileCompany 8`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
  <div>
    <h2>
      Новый клиент
    </h2>
    <label
      className="id"
    >
      <b>
        ID:
      </b>
      102
    </label>
     
    <br />
    <label
      htmlFor="LastName"
    >
      <b>
        Фамилия:
      </b>
    </label>
     
    <input
      id="LastName"
      type="text"
    />
     
    <br />
    <label
      htmlFor="Name"
    >
      <b>
        Имя:
      </b>
    </label>
     
    <input
      id="Name"
      type="text"
    />
     
    <br />
    <label
      htmlFor="MiddleName"
    >
      <b>
        Отчество:
      </b>
    </label>
     
    <input
      id="MiddleName"
      type="text"
    />
     
    <br />
    <label
      htmlFor="Balance"
    >
      <b>
        Баланс:
      </b>
    </label>
     
    <input
      id="Balance"
      type="number"
    />
    <br />
    <input
      defaultValue="Добавить"
      onClick={[Function]}
      type="button"
    />
    <input
      defaultValue="Закончить"
      onClick={[Function]}
      type="button"
    />
  </div>
</div>
`;

exports[`работа MobileCompany 9`] = `
<div
  className="MobileCompany"
>
  <hr />
  <input
    bb1="block"
    onClick={[Function]}
    type="button"
    value="Все"
  />
  <input
    bb2="block"
    onClick={[Function]}
    type="button"
    value="Активные"
  />
  <input
    bb3="block"
    onClick={[Function]}
    type="button"
    value="Заблокированные"
  />
  <hr />
  <table
    className="MobileCompanyClients"
  >
    <tbody>
      <tr>
        <th>
          Фамилия
        </th>
        <th>
          Имя
        </th>
        <th>
          Отчество
        </th>
        <th>
          Статус
        </th>
        <th>
          Редактировать
        </th>
        <th>
          Удалить
        </th>
      </tr>
      <tr
        className="MobileClient"
      >
        <td
          className="MobileClientFIO"
        >
          Григорьев 
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорий
        </td>
        <td
          className="MobileClientFIO"
        >
          Григорьевич
        </td>
        <td
          className="MobileClientBalance"
        >
          -220
        </td>
        <td
          className="blocked"
        >
          blocked
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="5"
            onClick={[Function]}
            type="button"
            value="Редактировать"
          />
        </td>
        <td
          className="MobileClientFIO"
        >
          <input
            id="6"
            onClick={[Function]}
            type="button"
            value="Удалить"
          />
        </td>
      </tr>
    </tbody>
  </table>
  <input
    bb4="block"
    id="4"
    onClick={[Function]}
    type="button"
    value="Добавить клиента"
  />
  <div>
    <h2>
      Новый клиент
    </h2>
    <label
      className="id"
    >
      <b>
        ID:
      </b>
      102
    </label>
     
    <br />
    <label
      htmlFor="LastName"
    >
      <b>
        Фамилия:
      </b>
    </label>
     
    <input
      id="LastName"
      type="text"
    />
     
    <br />
    <label
      htmlFor="Name"
    >
      <b>
        Имя:
      </b>
    </label>
     
    <input
      id="Name"
      type="text"
    />
     
    <br />
    <label
      htmlFor="MiddleName"
    >
      <b>
        Отчество:
      </b>
    </label>
     
    <input
      id="MiddleName"
      type="text"
    />
     
    <br />
    <label
      htmlFor="Balance"
    >
      <b>
        Баланс:
      </b>
    </label>
     
    <input
      id="Balance"
      type="number"
    />
    <br />
    <input
      defaultValue="Добавить"
      onClick={[Function]}
      type="button"
    />
    <input
      defaultValue="Закончить"
      onClick={[Function]}
      type="button"
    />
  </div>
</div>
`;
